python/tank/platform/engine.py

```diff
@@ -169,6 +171,13 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt.QtGui = base_def.get("qt_gui")
         qt.TankDialogBase = base_def.get("dialog_base")
 
+        qt5_root = self.__define_qt5()
```

Comment:
should this be in `util.qt_importer` rather than the engine? thnking it may be nice for encapusluation and code structure purposes. Also, someone might wanna grab that module and use it for other things. And we may have cases like the login window, where we need it outside the engine!